servicemesh-install/topics/appendix-threescale-adapter.adoc

```diff
@@ -21,7 +22,32 @@ You can use these examples to configure requests to your services using the 3sca
 Pay particular attention to the `kind: handler` resource. You must update this with your 3scale credentials and the service ID of the API you want to manage.
 ====
 
-Modify the handler configuration with your 3scale configuration.
+. Modify the handler configuration with your 3scale configuration.
++
+[source,yaml]
+----
+  apiVersion: "config.istio.io/v1alpha2"
+  kind: handler
+  metadata:
+   name: threescale
+   namespace: istio-system
+  spec:
+   adapter: threescale
+   params:
+     service_id: "<SERVICE_ID>"
+     system_url: "https://<organization>-admin.3scale.net/"
+     access_token: "<ACCESS_TOKEN>"
+   connection:
+     address: "threescale-istio-adapter:3333"
+----
+
++
+. Create the required resources:
++
+----
+oc create -f istio/ -n istio-system
```

Comment:
This will vary depending on where the controlplane is installed.

Reply:
@brian-avery up until now the Maistra docs had been pointing towards `istio-system` namespace I believe?? If that has changed then good to update and add more details here also @dfennessy

Reply:
With this release, we're adding support for multitenancy. In a single control plane mode everything will be in Istio-system as expected in previous releases. In multitenancy, the user can have as many control planes as they want and each one of the control planes has a different control plane namespace.

Reply:
@brian-avery @dfennessy I take it the 3scale team needs to factor multi-tenancy into their documentation.

Reply:
Correct, we will need to update that upstream ASAP. I've created an issue

Reply:
@dfennessy - IMO for now a note saying modify the namespace in the resources accordingly should be enough, wdyt?

Reply:
Agreed @PhilipGough

Reply:
I'd remove the namespace from the resource (`namespace: istio-system`) and just use the `-n` flag on the `oc` call.

Reply:
+1 to Rob's suggestion, even in a single cluster wide installation it's not necessarily going to be in istio-system

Reply:
+1 sounds good

Reply:
I've made these changes and updated the PR.